### PR TITLE
add routes for tool i18n

### PR DESCRIPTION
**Pull request by benib**

### CODE REVIEW

File: routes/editor/locales.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      const tool = tools[toolName];
+      if (!tool.editor.hasOwnProperty('label_locales') || !tool.editor.label_locales.hasOwnProperty(request.params.lng)) {
+        continue;
+      }
+      translations[toolName] = tool.editor.label_locales[request.params.lng];
+    }
+
+    reply(translations);
+  },
+  config: {
```
Comment (fuenkchen) [inline routes/editor/locales.js]: Maybe move config above handler: the majority of routes is defined in that order. Would be nice to have the same structure in the routes, so we catch the properties of each route quicker.

File: routes/tools/locales.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  handler: function(request, reply) {
+    let lng = request.params.lng;    
+    request.server.methods.getTranslations(request.params.tool, lng, (err, result) => {
+      if (err) {
+        return reply(err);
+      }
+      return reply(result);
+    })
+  },
+  config: {
```
Comment (fuenkchen) [inline routes/tools/locales.js]: same as in `routes/editor/locales.js`

Comment (benib) [inline routes/tools/locales.js]: agree, it was not consistent at all. I changed it now for all routes, which bloats this PR slightly, but gives us consistency regarding the order.

Comment (fuenkchen) [inline routes/tools/locales.js]: perfect. if we have a similar issue in the routes of the tools, we can correct it whenever we see it.

**Pull request merged**